indico/modules/events/management/templates/event_person_list.html

```diff
@@ -54,7 +54,7 @@
                         <td class="i-table">
                             {% if person_roles.primary_author or person_roles.secondary_author %}
                                 {% set author_types = [_('Primary') if person_roles.primary_author,
-                                                       _('Secondary') if person_roles.secondary_author] %}
+                                                       _('Co-author') if person_roles.secondary_author] %}
```

Comment:
If someone has a good idea on how to express the author type (none/primary/co-author) using icons it would probably look better than the text